ibis/backends/bigquery/compiler.py

```diff
@@ -65,8 +66,7 @@ def find_bigquery_udf(op):
     return lin.proceed, result
 
 
-_NAME_REGEX = re.compile(r'[^!"$()*,./;?@[\\\]^`{}~\n]+')
-_EXACT_NAME_REGEX = re.compile(f"^{_NAME_REGEX.pattern}$")
+_NAME_REGEX = re.compile(r'[^!"$()*,./;?@[\\\]^`{}~\n\s]+')
```

Comment:
What's the reason for the addition of `\s`? Spaces and tabs should be allowed.

Reply:
Let me see if I can revert this. I think this was from a previous attempt.

Reply:
Reverted!